codec/common/src/mc.cpp

```diff
@@ -4195,6 +4196,189 @@ void PixelAvg_mmi(uint8_t* pDst, int32_t iDstStride, const uint8_t* pSrcA, int32
   kpfFuncs[iWidth >> 4] (pDst, iDstStride, pSrcA, iSrcAStride, pSrcB, iSrcBStride, iHeight);
 }
 #endif//HAVE_MMI
+
+#if defined(HAVE_LSX)
+
+void McCopyWidthEq4_lsx(const uint8_t *pSrc, int iSrcStride,
```

Comment:
it is recommend move all the SIMD/intrinsic functions to a separate file.

Reply:
Yeah, that's better. Will update.